From 6c4267ca5bc2875779b59a945ec9eefbb2093801 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Fri, 1 Apr 2022 16:35:32 -0300
Subject: [PATCH 157/163] math: Fix isgreater* and isless* for clang

clang does not check for unordered numbers with builtins for
_Float128 type.
---
 math/math.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/math/math.h b/math/math.h
index 27963ef6dc..ce830ac1fa 100644
--- a/math/math.h
+++ b/math/math.h
@@ -1295,7 +1295,7 @@ iszero (__T __val)
 #endif
 
 #ifdef __USE_ISOC99
-# if __GNUC_PREREQ (3, 1)
+# if __GNUC_PREREQ (3, 1) && !defined __clang__
 /* ISO C99 defines some macros to compare number while taking care for
    unordered numbers.  Many FPUs provide special instructions to support
    these operations.  Generic support in GCC for these as builtins went
-- 
2.37.2

